demo/temp/gbe-all-plugins-bundle.js

```diff
@@ -10401,7 +10401,7 @@ function plural(ms, n, name) {
 /* WEBPACK VAR INJECTION */(function(global) {/*!
```

Comment:
Especially since we're so close to releasing things, we don't want to be introducing changes to this bundle. Please revert.